### ISSUE TITLE
Dnspy ignore errors .

### ISSUE THREAD

**Issue by marcocorriero** [0xd4d/dnSpy#1369]
Before clicking `Submit new issue`, try the latest build [![](https://github.com/0xd4d/dnSpy/workflows/GitHub%20CI/badge.svg)](https://github.com/0xd4d/dnSpy/actions)

- What version did you use?
Latest
- Write down all steps needed to reproduce this issue. Also attach any needed files.
None

sometimes when i edit the classes and i do something idk , tons of windows pop out, even if i close one another one appears, making dnspy unusable, maybe add something to hide these windows errors?

**Comment by 0xd4d** [comment]
Can you show a picture of these windows?

**Comment by marcocorriero** [comment]
> Can you show a picture of these windows?

 If the error happens again, ill make sure to do a screenshot and put here, but for now is going fine, im pretty sure i can make it happen.

**Comment by marcocorriero** [comment]
@0xd4d i can't reopen this issue, but i got blocked by this error again

https://i.imgur.com/RMSjf1B.png

It keeps reopening.

**Comment by 0xd4d** [comment]
I've added a possible fix. The line numbers don't match so I don't know if it's the correct fix.

You can get binaries once the CI has built it: https://github.com/0xd4d/dnSpy/actions/runs/38313996
